featurebyte/service/online_enable.py

```diff
@@ -6,9 +6,7 @@
 from typing import Any, Optional
 
 from bson.objectid import ObjectId
-from pydantic import PrivateAttr
 
-from featurebyte.feature_manager.manager import FeatureManager
 from featurebyte.feature_manager.model import ExtendedFeatureModel
```

Comment:
👍 Good to move `FeatureManager` to service directory. Would be good if we can also move the `ExtendedFeatureModel` from `featurebyte.feature_manager.model` to `featurebyte.models` (definitely should do it in another PR).